grpc/ca-wrappers.go

```diff
@@ -82,14 +82,14 @@ func NewCertificateAuthorityServer(inner core.CertificateAuthority) *Certificate
 }
 
 func (cas *CertificateAuthorityServerWrapper) IssuePrecertificate(ctx context.Context, request *capb.IssueCertificateRequest) (*capb.IssuePrecertificateResponse, error) {
-	if request == nil || request.Csr == nil || request.OrderID == nil || request.RegistrationID == nil {
+	if request == nil || request.Csr == nil {
```

Comment:
In some cases like this, I have left a check in for `.Csr` because that's a []bytes type, which can theoretically still be nil.